pynn_spinnaker/simulator.py

```diff
@@ -112,6 +112,14 @@ def _wait_for_transition(self, placements, allocations,
             raise Exception("Unexpected core failures "
                             "before reaching %s state (%u/%u)." % (to_state, cores_in_to_state, num_verts))
 
+    def _configure_software_watchdog(self):
+        # If software watchdog is disabled, write zero to machine's
+        # SV struct, otherwise, write default from sv struct file
+        # **NOTE** x, y and p should already be in context
+        w = (0 if self.disable_software_watchdog
+             else self.machine_controller.structs["sv"]["soft_wdog"].default)
+        self.machine_controller.write_struct_field("sv", "soft_wdog", w)
```

Comment:
It wouldn't be SC&MP without undocumented magic numbers... After delving in: The number is 0 to disable the watchdog and otherwise gives the number of consecutive "pings" sent to a core that must go missing before killing the core. Cores are pung at 10ms intervals so essentially soft_wdog is the number of 10ms intervals a core must be unresponsive for before it gets killed. A core gets 1ms to respond to a ping before it times out.